rse-publish.Rmd

```diff
@@ -0,0 +1,544 @@
+# Publishing {#publish}
+
+## Questions {#publish-questions}
+
+```{r, child="questions/publish.md"}
+```
+
+## Objectives {#publish-objectives}
+
+```{r, child="objectives/publish.md"}
+```
+
+## Introduction {#publish-intro}
+
+This lesson looks at what should be included in reports and how best to do that.
+We use the generic term "report" to include research papers,
+summaries for clients,
+and everything else that is shorter than a book and is going to be read by someone else.
+
+Our motivation is summed up in this quotation:
+
+> An article about computational science in a scientific publication is *not* the scholarship itself,
+> it is merely *advertising* of the scholarship.
+> The actual scholarship is the complete software development environment
+> and the complete set of instructions which generated the figures.
+>
+> -- Jonathan Buckheit and David Donoho, paraphrasing Jon Claerbout, in @Buck1995
+
+As the quote suggests,
+modern publishing involves much more than simply producing a written report.
+It involves providing readers with the data underpinning the report,
+as well as any code written in analysing the data.
+
+While the definition of data (and its associated metadata)
+is relatively easy to understand,
+code can come in many different forms.
+Here we distinguish between “analysis code” written
+solely for the purpose of the report (e.g. to produce a figure)
+and “analysis software” that is formally packaged and released for use by a wider audience.
```

Comment:
Mm - maybe "script" rather than "code"? There's gotta be better terminology somewhere ...

Reply:
I've only ever seen use of the terms software and code (e.g. [example](https://www.nature.com/articles/d41586-018-02741-4)), but it's not like there's an accepted, widely used language. I like "analysis scripts" better than "analysis code", so I'll swtich to that for now.